### PR TITLE
Use complete pokedex for production calculator

### PR DESCRIPTION
**Pull request by 3geek14**
Add all Pokemon to production calculator

Also alphebetize the list, and use the in-game names for event Pokemon.

### CODE REVIEW

File: backend/src/utils/pokemon-utils/pokemon-utils.ts
Comment (SleepAPI) [inline backend/src/utils/pokemon-utils/pokemon-utils.ts:1]: I'm not sure what prompted this change, while it's fine, it's also not needed for this PR.
Were you trying to solve #27 at the same time perhaps? If so, that would be the function below `getPokemonNames`. Then you would need to update the tests too and while you're at it getPokemonForName doesn't appear to be covered by tests.

I'm also fine with doing this in a different PR though, just mentioned it as I was unsure what prompted this change.

**Review by SleepAPI — COMMENTED** [review]
Let one comment to check out, but everything looks great to me!
I'll approve after backend is merged, just to see that this PR is updated correctly accordingly

**Comment by 3geek14** [comment]
Removed the extra file and the association with #27. Tested locally that the production calculator shows added Pokemon in the dropdown.

**Review by SleepAPI — APPROVED** [review]
Looks great to me! Wonderful contribution!

**Pull request merged**
